providers/base/units/audio/manifest.pxu

```diff
@@ -4,16 +4,6 @@
 # Written by:
 #   Sylvain Pineau <sylvain.pineau@canonical.com>
 
-unit: manifest entry
-id: has_audio_playback
-_name: Audio playback
-value-type: bool
-
-unit: manifest entry
-id: has_audio_capture
-_name: Audio capture
-value-type: bool
```

Comment:
As discussed above, it might be better to keep manual manifest entries to let the tester tell Checkbox if the device being tested has sound input/output or not.